### PR TITLE
Adjust lava damage values

### PR DESCRIPTION
**Pull request by bearbin**
Fixes issue #2925

### CODE REVIEW

File: src/Entities/Entity.cpp [hunk 1]
```diff
@@ -1293,7 +1293,7 @@ void cEntity::TickBurning(cChunk & a_Chunk)
 		m_TicksSinceLastFireDamage++;
 		if (m_TicksSinceLastFireDamage >= FIRE_TICKS_PER_DAMAGE)
 		{
-			if (!IsFireproof())
+			if (!(IsFireproof() || HasLava))
```
Comment (madmaxoft) [inline src/Entities/Entity.cpp]: I think the DeMorgan-ed version would be a bit easier to understand:
```cpp
if (!IsFireproof() && !HasLava)  // Not fireproof, and hasn't taken lava-damage already (#2197)
```

Comment (bearbin) [reply src/Entities/Entity.cpp]: Fixed.

**Comment by bearbin** [comment]
The travis bulild actually passed, it just errored when downloading the packages.

**Comment by madmaxoft** [comment]
Then the easiest thing to do is to restart the job, you should have the permissions for that. I did it for you now.

**Pull request merged**